jxl-oxide 0.12.5 doc build failed

https://docs.rs/crate/jxl-oxide/latest/builds/2548932

```
[INFO] [stderr] warning: target filter specified, but no targets matched; this is a no-op
[INFO] [stderr]  Documenting jxl-oxide v0.12.5 (/opt/rustwide/workdir)
[INFO] [stderr] error[E0557]: feature has been removed
[INFO] [stderr]    --> src/lib.rs:148:29
[INFO] [stderr]     |
[INFO] [stderr] 148 | #![cfg_attr(docsrs, feature(doc_auto_cfg))]
[INFO] [stderr]     |                             ^^^^^^^^^^^^ feature has been removed
[INFO] [stderr]     |
[INFO] [stderr]     = note: removed in CURRENT_RUSTC_VERSION; see <https://github.com/rust-lang/rust/pull/138907> for more information
[INFO] [stderr]     = note: merged into `doc_cfg`
[INFO] [stderr] 
[INFO] [stderr] error: Compilation failed, aborting rustdoc
[INFO] [stderr] 
[INFO] [stderr] For more information about this error, try `rustc --explain E0557`.
[INFO] [stderr] error: could not document `jxl-oxide`
```

--- a/crates/jxl-oxide/src/lib.rs
+++ b/crates/jxl-oxide/src/lib.rs
@@ -145,7 +145,7 @@
 //! - `lcms2`: Enable integration with Little CMS 2.
 //! - `moxcms`: Enable integration with `moxcms` crate.
 
-#![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg))]
 
 use std::sync::Arc;
 

